packages/node_modules/@ciscospark/internal-plugin-metrics/src/metrics.js

```diff
@@ -55,9 +67,12 @@ const Metrics = SparkPlugin.extend({
       // their metrics will not post from a queue flush in time
       return this.postPreLoginMetric(_payload, preLoginId);
     }
-    return this.clientMetricsBatcher.request(payload);
+    return this.sendBatchMetric(payload);
   },
 
+  sendBatchMetric(payload) {
```

Comment:
This should probably be prefixed with an `_`.

Reply:
done